ui/components/app/nfts-tab/nfts-tab.js

```diff
@@ -88,137 +104,112 @@ export default function NftsTab() {
   }
 
   return (
-    <Box className="nfts-tab">
-      {hasAnyNfts > 0 || previouslyOwnedCollection.nfts.length > 0 ? (
-        <NftsItems
-          collections={collections}
-          previouslyOwnedCollection={previouslyOwnedCollection}
-        />
-      ) : (
-        <>
-          {isMainnet && !useNftDetection ? (
-            <Box paddingTop={4} paddingInlineStart={4} paddingInlineEnd={4}>
-              <NFTsDetectionNoticeNFTsTab />
-            </Box>
-          ) : null}
-          {
-            ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
-            showNftBanner ? (
-              <Box
-                paddingInlineStart={4}
-                paddingInlineEnd={4}
-                display={Display.Flex}
-                paddingTop={4}
-              >
-                <AssetListConversionButton
-                  variant={ASSET_LIST_CONVERSION_BUTTON_VARIANT_TYPES.NFT}
-                  onClick={() => {
-                    global.platform.openTab({ url: ZENDESK_URLS.NFT_TOKENS });
-                    trackEvent({
-                      event: MetaMetricsEventName.EmptyNftsBannerClicked,
-                      properties: {
-                        chain_id: chainId,
-                        locale: currentLocale,
-                        network: nickname,
-                        referrer: ORIGIN_METAMASK,
-                      },
-                    });
-                  }}
-                />
+    <>
+      {showRampsCard && <RampsCard variant={RAMPS_CARD_VARIANT_TYPES.NFT} />}
```

Comment:
Same comment as before. 
1. Prefer ternary and null as previous code was
2. the `showNftBanner` ternary was fenced by `///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)`, we should keep the behavior here

```suggestion
      {
        ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
        showRampsCard ? (
          <RampsCard variant={RAMPS_CARD_VARIANT_TYPES.NFT} />
        ): null
        ///: END:ONLY_INCLUDE_IF
      }
```

This should be applied to the imports as well.

Reply:
[40eb114](https://github.com/MetaMask/metamask-extension/pull/23352/commits/40eb114da411c8685f3f6e32479c388723bc032e)